test/README.md

```diff
@@ -107,6 +107,33 @@ how many jobs to run, append `--jobs=n`
 The individual tests and the test_runner harness have many command-line
 options. Run `test/functional/test_runner.py -h` to see them all.
 
+#### Speed up test runs with a ramdisk
+
+If you have some available RAM on your system you can create a ramdisk to use as the `cache` and `tmp` directories for the functional tests in order to speed them up.
+Speed-up amount varies on each system (and according to your ram speed and other variables), but a 2-3x speed-up is not uncommon.
+
+To create a 4GB ramdisk at `/mnt/tmp/`:
+
+```bash
+sudo mkdir -p /mnt/tmp
+sudo mount -t tmpfs -o size=4g tmpfs /mnt/tmp/
+```
+
+Configure the size of the ramdisk using the `size=` option.
+A full test suite run with `--no-cleanup` takes slightly over 8GB of disk space in total, so configuring above this seems unnecessary, however even smaller ramdisks have a great impact on test suite speed.
```

Comment:
This should be `--nocleanup`

```suggestion
A full test suite run with `--nocleanup` takes slightly over 8GB of disk space in total, so configuring above this seems unnecessary, however even smaller ramdisks have a great impact on test suite speed.
```

Reply:
Updated (removed) as kind of irrelevant to the normal usage.